src/Distortion.cpp

```diff
@@ -173,6 +158,47 @@ void removeDistortion(double dx, double dy, double &ux, double &uy,
       // number of iterations
     }
     break;
+    case KAGUYATC: {
+      // Apply distortion correction
+      // see: SEL_TC_V01.TI
+      // r2 = x^2 + y^2
+      //   Line-of-sight vector of pixel no. n can be expressed as below.
+
+      //  Distortion coefficients information:
+      //   INS<INSTID>_DISTORTION_COEF_X  = ( a0, a1, a2, a3)
+      //   INS<INSTID>_DISTORTION_COEF_Y  = ( b0, b1, b2, b3),
+      //
+      // Distance r from the center:
+      //   r = - (n - INS<INSTID>_CENTER) * INS<INSTID>_PIXEL_SIZE.
+      //
+      // Line-of-sight vector v is calculated as
+      //   v[X] = INS<INSTID>BORESIGHT[X]
+      //          +a0 +a1*r +a2*r^2 +a3*r^3 ,
+      //   v[Y] = INS<INSTID>BORESIGHT[Y]
+      //          +r +a0 +a1*r +a2*r^2 +a3*r^3 ,
```

Comment:
I need to change this in the comments here in ISIS, too. After much discussion, we ended up deciding that the second equation should actually be: 

```
//   v[Y] = INS<INSTID>BORESIGHT[Y]
//           b0 +b1*r +b2*r^2 +b3*r^3 
```
so the comments should probably be updated with that.